### ISSUE TITLE
Optimal parameters are gone when del_stressmodel is run twice

### ISSUE THREAD

**Issue by rubencalje** [pastas/pastas#84]
When del_transform is ran twice, for example

```
ml.del_transform('recharge')
ml.del_transform('well 1') 
```
you lose the optimal parameters. After the first time the initial parameters are set to the optimal parameters, and after the second time the initial parameters are overwitten by the default initial parameters.

**Comment by raoulcollenteur** [comment]
example given should be:
```
ml.del_stressmodel('recharge')
ml.del_stressmodel('well 1') 
```

solution can be to copy values to optimal parameters too.

**Comment by rubencalje** [comment]
eh, yes, I mean del_stressmodel.

But we need Nans in the optimal parameters to notify the user that the new set of parameters has not been optimized yet, right? So we need a new boolean attribute of the model that indicates if a model has been solved allready:

`ml.solved = True`

**Comment by raoulcollenteur** [comment]
Similar things happen when add_stressmodel is used. get_init_parameters should be called with initial=False

**Comment by raoulcollenteur** [comment]
This issue has been solved as per commit 8da3745 . Closing this issue.
